src/karma_tracker/api/core.clj

```diff
@@ -15,15 +16,16 @@
       (wrap-cors :access-control-allow-origin [#".*"]
                  :access-control-allow-methods [:get])))
 
-(defn api [execution-fns]
-  (-> (routes
-       (query-routes (:query execution-fns)))
-      wrap-handler))
-
 (defn get-execution-fns [resources]
   {:query (partial query/execute resources)})
 
+(defn api [resources]
+  (-> (routes
+       (health-check-routes resources)
+       (query-routes (-> resources get-execution-fns :query)))
+      wrap-handler))
+
 (defn run-server [resources]
   (println "Api server running on port 8000")
-  (run-jetty (api (get-execution-fns resources))
+  (run-jetty (api resources)
```

Comment:
yeah thanks could you please make the changes you need. the health check endpoint needs access to mongo so that will need to be refactored